### PR TITLE
Fix comment form styling

### PR DESCRIPTION
**Pull request by kienstra**
**Request For Review**

As @ThierryA mentioned, there were styling issues on the single 'adventure' page's comment form:
* The "Name" and "Email" should appear next to each other on desktop displays
* There wasn't enough space between some `<label>` and `<input>` elements

https://dev-amp-start-travel.pantheonsite.io/adventure/surfing-day-board-and-wetsuits-included-in-price/

adventure-page

Here's the display with this PR applied:
single-adventure-pr-applied

### CODE REVIEW

File: assets/css/adventure.css [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
...object-cover,.travel-object-cover>img{-o-object-fit:cover;object-fit:cover}amp-img.rounded img{border-radius:4px}amp-img.circle img{border-radius:50%}.travel-no-focus:focus{outline:none}.travel-type-active,.travel-type-active:visited{color:#fd68ae}.travel-type-artistic,.travel-type-artistic:visited{color:#32dada}.travel-type-drinks,.travel-type-drinks:visited{color:#4e93f8}.travel-type-fashion,.travel-type-fashion:visited{color:#f1c363}.travel-type-food,.travel-type-food:visited{color:#aa5efd}.travel-type-music,.travel-type-music:visited{color:#ec8936}.travel-type-nature,.travel-type-nature:visited{color:#14b52a}.travel-type-nightlife,.travel-type-nightlife:visited{color:#e83a3a}.travel-type-tours,.travel-type-tours:visited{color:#0ece9e}.travel-type-water,.travel-type-water:visited{color:#3abee8}.travel-overflow-container{display:inline-block;min-width:100vw;padding-left:calc((100vw - 72rem) / 2);padding-right:calc((100vw - 72rem) / 2)}.travel-overflow-container>.travel-overflow-wrap{max-width:75rem;margin:0 auto}.travel-link,.travel-link:visited{color:#8b58e3;text-decoration:none;font-family:inherit;-webkit-transition:color .15s ease-in;transition:color .15s ease-in}.travel-link:
```
Comment (kienstra) [inline assets/css/adventure.css]: This diff looks more extensive than it is. I only removed `.comment-form .comment-form-email`
from before the rule:
```css
.comment-form .comment-form-rating{width:100%;padding-right:15px;margin-bottom:-15px;}
```

Comment (kienstra) [inline assets/css/adventure.css]: This is simply copied from an earlier version of this file. 
(This comment is intended for the line that begins with `@media (min-width: 52.06rem) {`,)

**Comment by DavidCramer** [comment]
Hi @kienstra Just one issue and we're 100's!
image 2018-05-07 at 6 55 46 pm
Easy solution:...
image 2018-05-07 at 6 57 27 pm

**Review by DavidCramer — CHANGES_REQUESTED** [review]
Just that email field padding on the right and we're good!

**Review by DavidCramer — APPROVED** [review]

**Comment by DavidCramer** [comment]
Looks good to me! Thanks @kienstra

**Comment by kienstra** [comment]
@DavidCramer, thanks for reviewing this!

**Pull request merged**